site/pages/about/roadmap.tsx

```diff
@@ -1,68 +1,196 @@
+            )}
+            showSeparator
+          >
+            <UnorderedList
+              overrides={{
+                content: {
+                  typographyPreset: 'editorialParagraph020',
+                  stylePreset: 'inkBase',
+                },
+              }}
+            >
+              {cmsData
+                .filter(data => data[0].startsWith('current_li'))
+                .map(entry => (
+                  <FormatMarkdown>{entry[1]}</FormatMarkdown>
+                ))}
+            </UnorderedList>
+          </ContentPrimary>
+        </ContentSection>
+        <ContentSection sectionName="Coming up">
+          <ContentPrimary
+            id="overview"
+            toc={getValueFromCMS(content, 'comingup_headline', 'Coming Up')}
+            headline={getValueFromCMS(
+              content,
+              'comingup_headline',
+              'Coming Up',
+            )}
+            description={getValueFromCMS(
+              content,
+              'comingup_description',
+              'The focus for the next quarter:',
+            )}
+            showSeparator
+          >
+            <UnorderedList
+              overrides={{
+                content: {
+                  typographyPreset: 'editorialParagraph020',
+                  stylePreset: 'inkBase',
+                },
+              }}
+            >
+              {cmsData
+                .filter(data => data[0].startsWith('comingup_li'))
+                .map(entry => (
+                  <FormatMarkdown>{entry[1]}</FormatMarkdown>
+                ))}
+            </UnorderedList>
+          </ContentPrimary>
+        </ContentSection>
+        <ContentSection sectionName="Future">
+          <ContentPrimary
+            id="overview"
+            toc={getValueFromCMS(content, 'future_headline', 'Future')}
+            headline={getValueFromCMS(content, 'future_headline', 'Future')}
+            description={getValueFromCMS(
+              content,
+              'future_description',
+              'Ideas we plan to look at:',
+            )}
+            showSeparator
+          >
+            <UnorderedList
+              overrides={{
+                content: {
+                  typographyPreset: 'editorialParagraph020',
+                  stylePreset: 'inkBase',
+                },
+              }}
+            >
+              {cmsData
+                .filter(cells => cells[0].startsWith('future_li'))
+                .map(entry => (
+                  <FormatMarkdown>{entry[1]}</FormatMarkdown>
+                ))}
+            </UnorderedList>
+          </ContentPrimary>
+        </ContentSection>
+      </ComponentPageCell>
+    </AboutPageTemplate>
+  );
+};
 
 export default Roadmap;
+
+// This function is called at build time and the response is passed to the page
+// component as props.
+export async function getStaticProps() {
+  const [cmsData] = await Promise.all([getSheets('Roadmap')]);
```

Comment:
In this case, we have only 1 async task so no promise.all is needed
```suggestion
  const cmsData = await getSheets('Roadmap');
```